airflow/exceptions.py

```diff
@@ -395,6 +395,19 @@ class ConnectionNotUnique(AirflowException):
     """Raise when multiple values are found for the same connection ID."""
 
 
+class SkipDownstreamTaskInstances(BaseException):
```

Comment:
Should I relocate this exception to the SDK? (the other exceptions here are also used there)

Reply:
Yes, may as well do that now since it's a new one.

Reply:
Became irrelevant